benchmark/python/benchmark_e2e.py

```diff
@@ -0,0 +1,201 @@
+# This is an end-to-end benchmarking script for any ONNX model.
+#
+# Prerequisites: 
+# 0) Install onnxruntime-genai and onnxruntime
+#
+# 1) Use builder.py to build the desired ONNX model
+#
+# 2) For accurate wall clock numbers, modify genai_config.json to include the following:
+#    "min_length": {prompt_length + generation_length}
```

Comment:
does the user need to specify the min_length? I think we can specify it in the code